### PR TITLE
Mobile compatability

### PR DESCRIPTION
**Pull request by lzoebaker**
To make the site more compatable with mobile devices, I used a media query to test for the device width. If the device is mobile, the website:
* uses a static background image
* no longer has a button/ button description to change background image
* instead of hovering over the profile icon to change it to the gif, must actively click (hover does not work on mobile devices)

Here is the preview on mobile devices:
IMG_8425
IMG_8426

### CODE REVIEW

File: portfolio/src/main/webapp/jquery_functions.js [hunk 1]
```diff
@@ -2,40 +2,39 @@
...
 $(document).ready(function(){
+  let isMobile = detectMobileDevice();
+
+  let iconImages = preloadImages(profileImagePaths);
+ 
+  if (!isMobile){
+    // if change background button is clicked, background image for the body is changed.
+    $("#change-background").click(function(){
+      updateBackground(currentImageIndex);
+     currentImageIndex = (currentImageIndex + 1) % NUM_IMAGES;
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/jquery_functions.js:21]: indentation seems off

File: portfolio/src/main/webapp/jquery_functions.js [hunk 2]
```diff
@@ -2,40 +2,39 @@
...
     function(){
         $(".profile").attr("src", "images/zoebakercircle.png")
     });
-
+  } else { // if mobile, actively click to change (hover does not work as expected)
+    $(".profile").toggle(function(){
+      $(".profile").attr("src", "images/noogler.gif")
+    },
+    function(){
+      $(".profile").attr("src", "images/zoebakercircle.png")
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/jquery_functions.js:37]: You use these strings in multiple places now. Consider storing them in a constant variable

File: portfolio/src/main/webapp/jquery_functions.js [hunk 3]
```diff
@@ -2,40 +2,39 @@
...
-    image_arr[i] = temp_image;
-  }
-  return image_arr;
-}
-
-let current_image_index = 1; // start on second image so as not to repeat landing image
 $(document).ready(function(){
+  let isMobile = detectMobileDevice();
+
+  let iconImages = preloadImages(profileImagePaths);
```
Comment (WillHipschman) [inline portfolio/src/main/webapp/jquery_functions.js:15]: Do we need to preloadImages on mobile? They can't change the background right? Doesn't this make them load more data then necessary on a mobile device?

Comment (lzoebaker) [reply portfolio/src/main/webapp/jquery_functions.js:15]: For this case, preloadImages is used only for changing the icon photo (not the background), and that functionality is implemented in mobile.

File: portfolio/src/main/webapp/jquery_functions.js (hunk 1 above)
Comment (lzoebaker) [reply portfolio/src/main/webapp/jquery_functions.js:21]: Good catch, changed.

File: portfolio/src/main/webapp/jquery_functions.js (hunk 2 above)
Comment (lzoebaker) [reply portfolio/src/main/webapp/jquery_functions.js:37]: Good idea.

**Pull request merged**